tests/providers/microsoft/azure/transfers/test_local_to_adls.py

```diff
@@ -28,11 +27,9 @@
 )
 
 TASK_ID = "test-adls-upload-operator"
-FILE_SYSTEM_NAME = "Fabric"
```

Comment:
why?

Reply:
Same reason, test moved from location, this line was added to test the new operator and is not used by the operator being tested here, it is now in the test file (tests/providers/microsoft/azure/operators/test_adls_create.py) of the new operator:

Reply:
The test_local_to_adls.py is now exactly the same as in main

Reply:
no...
You are requesting to remove this line.
https://github.com/apache/airflow/blob/354e633ddd7da5fb8310ef6bd1c5b9e5e8763848/tests/providers/microsoft/azure/transfers/test_local_to_adls.py#L31

Reply:
Indeed this line was also only used for testing new Operator, same logic as above, this is all due to moving the operator to another module (from transfers to operator), which meant moving tests also to corresponding operator module, so that constant ins't needed anymore here, its moved to test_adls_create.py now where the new operator is tested.